### DISCUSSION TITLE
[求助]绳网里面怎么提升自己的等级啊

### DISCUSSION THREAD

**Discussion by Cross-king** [share121/inter-knot#3457] [其他]
今天刚入绳网，看见一堆十几级的大佬，而我却等级零。在此求一个快速提升等级的方法，很急！qwq

**Comment by Guessme404** [comment]
成为一名优秀的程序猿就可以了，加油！<img src="https://img-static.mihoyo.com/communityweb/upload/408018316100a8a8a846c68acedaa191.gif" alt="米游姬 卖萌" width="75" >

**Comment by 1ctrl-cv** [comment]
多写几座史山就可以了![img](https://camo.githubusercontent.com/330ef8ead966011174d4cf253e9851f91ce32b2a70e86a4188ba15a9d1401977/68747470733a2f2f696d672d7374617469632e6d69686f796f2e636f6d2f636f6d6d756e6974797765622f75706c6f61642f34303830313833313631303061386138613834366336386163656461613139312e676966)

**Comment by CloudSwordSage** [comment]
建仓库，往里面丢代码，fork的仓库好像也算<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >
